code/modules/mob/living/carbon/human/ai/brain/ai_brain_guns.dm

```diff
@@ -36,30 +38,31 @@
 	set waitfor = FALSE
 
 	if(!primary_weapon || tried_reload)
-		currently_busy = FALSE
 		return
 
-	currently_busy = TRUE
+	currently_reloading = TRUE
```

Comment:
Are you sure this not setting currently busy won't have some knock on effects?

Reply:
Not so sure now i guess, let's add it